lib/task/migrate/migrations/arMigration0147.class.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * Add ability to set whether digital object geolocation shown
+ *
+ * @package    AccesstoMemory
+ * @subpackage migration
+ */
+class arMigration0147
+{
+  const
+    VERSION = 147, // The new database version
+    MIN_MILESTONE = 2; // The minimum milestone required
+
+  /**
+   * Upgrade
+   *
+   * @return bool True if the upgrade succeeded, False otherwise
+   */
+  public function up($configuration)
+  {
+    // Correct slug records created with a separator of . or / instead of -.
+    QubitPdo::prepareAndExecute('UPDATE slug SET slug = REPLACE(slug, ".", "-") WHERE slug LIKE "%.%"');
+    QubitPdo::prepareAndExecute('UPDATE slug SET slug = REPLACE(slug, "/", "-") WHERE slug LIKE "%/%"');
```

Comment:
That's cool. Should we chain the two REPLACE functions in one single query to avoid to passes to the entire table? Also, I've noticed that this issue has been tagged for v2.3.1 however I'm not sure if we're supposed to deliver new migrations in minor releases. Can you confirm that with @djjuhasz, please? If we did it, I think we'd need to reuse v139 and bump the others - it's too complicated.